libs/platform/table/components/table-row/table-row.component.ts

```diff
@@ -304,6 +305,24 @@ export class TableRowComponent<T> extends TableRowDirective implements OnInit, A
         return column.name;
     }
 
+    /** @hidden */
+    _itemFocusedEventAnnouncer = (position: FocusableItemPosition): string => {
+        let retVal = `Column ${position.colIndex + 1} of ${position.totalCols}, `;
```

Comment:
Shouldn't it be translatable via i18n?

Reply:
The whole thing needs to be translated, including what was there previously, so I figure that is a separate issue